integration/tests/api/event_test.go

```diff
@@ -52,24 +51,25 @@ func TestStreamingEventInformationIsReturned(t *testing.T) {
 		},
 	})
 
+	t.Logf("containerId in event_test is %v", containerId)
 	timeout := make(chan bool, 1)
 	go func() {
 		time.Sleep(60 * time.Second)
 		timeout <- true
 	}()
 
-	select {
-	case ev := <-einfo:
-		if ev.EventType == 0 {
-			marshaledData, err := json.Marshal(ev.EventData)
-			require.Nil(t, err)
-			var oomEvent *oomparser.OomInstance
-			err = json.Unmarshal(marshaledData, &oomEvent)
-			require.Nil(t, err)
-			require.True(t, oomEvent.ProcessName == "stress")
+	for {
+		select {
+		case ev := <-einfo:
+			if ev.EventType == 0 {
+				if strings.Trim(ev.ContainerName, "/ ") == strings.Trim("docker/"+containerId, "/ ") {
```

Comment:
this won't be true in CoreOS machines for example. WDYT of getting the name from a Client().ContainerInfo() and checking that?